content/team-current/milind.md

```diff
@@ -1,6 +1,6 @@
 +++
 
-date = "2018-06-24T00:00:00+05:30" 
+date = "2021-01-12T00:00:00+05:30" 
```

Comment:
@milindmantri 

Along with this, you need to set `is_alumnus = false`.

Refer to line no `9`.